### PR TITLE
Industry vertical misc changes

### PR DESCRIPTION
**Pull request by mjasrotia**
https://jira.corp.adobe.com/browse/PLAT-94658

### CODE REVIEW

**Review by hanessian — APPROVED** [review]

File: components/mixins/experience-event/experienceevent-offer-impression-details.schema.json [hunk 1]
```diff
@@ -25,24 +25,29 @@
     ]
   },
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
-  "description": "",
+  "description": "Offer details such as offer name, clicks, dismissals.",
   "definitions": {
     "offer-impression-details": {
       "properties": {
-        "xdm:marketing ": {
+        "xdm:offer ": {
```
Comment (hanessian) [inline components/mixins/experience-event/experienceevent-offer-impression-details.schema.json:32]: should this be offers so it stays consistent? Claims, Offers, etc?

File: components/mixins/experience-event/industry-verticals/experienceevent-bill-pay-details.schema.json [hunk 2]
```diff
@@ -12,9 +12,13 @@
...
-    "industry": ["financial_services"]
+    "industry": [
+      "financial_services"
+    ]
   },
-  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
   "description": "Bill pay steps such as start, save, failure, and complete.",
```
Comment (hanessian) [inline components/mixins/experience-event/industry-verticals/experienceevent-bill-pay-details.schema.json:22]: We need to update the description on this mixin now that the content has shifted dramatically. Let's put:
Bill pay details such as transaction information, service account ID, and balance.

File: components/mixins/profile/profile-claim-process.schema.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
     "you may not use this file except in compliance with the License. You may obtain a copy",
     "of the License at https://creativecommons.org/licenses/by/4.0/"
   ],
-  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-claim-process",
```
Comment (hanessian) [inline components/mixins/profile/profile-claim-process.schema.json:8]: @mjasrotia - I made this change later in the day so you may have missed it, but claim process shouldn't be moved to profile. It should stay as EE. We just need to remove all profile data nodes from the policy data type. 

1. Update top level grouping to be "claims"
2. The mixin should only have the following fields: policy ID, start date, status, type, and currency data type (named benefitAmount- see screenshot) 
image

**Review by hanessian — COMMENTED** [review]
Please see comments. Thanks!

File: components/mixins/experience-event/experienceevent-offer-impression-details.schema.json [hunk 4]
```diff
@@ -25,24 +25,29 @@
...
+        "xdm:offers ": {
           "type": "object",
           "properties": {
-            "xdm:offerClicks": {
+            "xdm:ID": {
+              "title": "Offer ID",
+              "type": "string",
+              "description": "Offer Identifier."
+            },
+            "xdm:clicks": {
```
Comment (prabhum2) [inline components/mixins/experience-event/experienceevent-offer-impression-details.schema.json:40]: how is this different from web.webinteraction.linkclicks ?

**Review by hanessian — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Review by cmathis — APPROVED** [review]

**Pull request merged**